### ISSUE TITLE
[Accessibility] The LocalizedControlType of ImageAlign/TextAlign/Anchor/Dock property is inconsistent in Inspect.

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#4102]
* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20513.14

* Have you experienced this same bug with .NET Framework?: 
No

* More Info:
This is a not regression issue, it occurs after issue [2074 ](https://github.com/dotnet/winforms/issues/2704)fixed.

**Problem description:**
The LocalizedControlType of following property is inconsistent in Inspect.
For **ImageAlign** & **TextAlign** property, it's **radio button**.
![ImageAlign](https://user-images.githubusercontent.com/62929087/95956879-83a8db80-0e31-11eb-9bca-6cc6188ac4e1.png)
For **Anchor** property, it's **panel**.
![Anchor](https://user-images.githubusercontent.com/62929087/95956927-94595180-0e31-11eb-8bc0-c1fc27a2715d.png)
For **Dock** property, it's **check box**.
![Dock](https://user-images.githubusercontent.com/62929087/95956938-991e0580-0e31-11eb-8955-49f5ebbc6cb1.png)

**Minimal repro:** 
[SampleApp.zip](https://github.com/dotnet/winforms/files/5376383/SampleApp.zip)
1. Create a winforms core app with PropertyGrid and Button control.
2. Associate the Button with PropertyGrid as a selected object.
3. Build and run the core project.
4. Start Inspect tool.
5. Open the drop down list next to ImageAlign.
6. Focus on Top Center and observe LocalizedControlType in Inspect.

**Comment by merriemcgaw** [comment]
So, for the three different custom drop downs, we wound up using 3 different control types 😵. This is not likely to block compliance for any application, so would not block WinForms compliance. I'll mark for accessibility backlog and future with that in mind.

**Comment by vladimir-krestov** [comment]
There are several ways to fix it easily:
- `ImageAlign` & `TextAlign`: they work like a RadioButton, so their control type should be "**radio button**"
   _How to fix_: do nothing, it's already "radio button"
- `Anchor`: it works like a CheckBox, so its control type should be "**check box**"
   _How to fix_: there are 2 ways:
     1) Override `Role` property for `SpringControlAccessibleObject` and set it as `CheckButton`:
![image](https://user-images.githubusercontent.com/49272759/124764964-2d8b4380-df3e-11eb-95c3-b193eada00ae.png)

     2) Set `AccessibleRole` of `SpringControl` as `CheckButton`:
![image](https://user-images.githubusercontent.com/49272759/124764799-02a0ef80-df3e-11eb-8fc4-dfdce5560bb0.png)

- `Dock`: it works like a RadioButton, so its control type should be "**radio butto**"
   _How to fix_: there are 2 ways:
     1) Set `AccessibleRole` of `DockEditorCheckBox` as `RadioButton`:
![image](https://user-images.githubusercontent.com/49272759/124765443-a2f71400-df3e-11eb-9f24-1aa7ad34e796.png)

     2) Change the base type of `DockEditorCheckBox` to `RadioButton`:
![image](https://user-images.githubusercontent.com/49272759/124765565-c28e3c80-df3e-11eb-8f2a-ed50e2865144.png)

@merriemcgaw, please check and approve the expected values of types.
I would assign it to @DmitryGorokhov.
@merriemcgaw, please approve before we start change this.

**Comment by merriemcgaw** [comment]
These all seem reasonable to me. I agree with your choices for Localized Control Type. @DmitryGorokhov, welcome aboard!

**Comment by Tanya-Solyanik** [comment]
![image](https://user-images.githubusercontent.com/15823268/125118616-ce940e80-e0a4-11eb-8d41-559527c0d083.png)

Why these 2 are ListItems?

Otherwise looks good. I would not change base type for the DockEditorCheckBox as there might be differences in renderers between these 2 classes

**Comment by vladimir-krestov** [comment]
It's just a mistake while testing the approach. Sorry

**Comment by John-Qiao** [comment]
Verified the issue in .NET 6.0.100-rc.1.21377.28 SDK from Master branch, this issue has been fixed that all LocalizedControlType are set correctly for ImageAlign & TextAlign, Anchor and Dock properties.
![ImageAlign-TextAlign](https://user-images.githubusercontent.com/45864985/127258246-5d354ac2-bc03-48ed-8613-1f36f7a9ae33.png)
![Anchor](https://user-images.githubusercontent.com/45864985/127258254-f02fff40-fc0c-486e-8a94-869b13db0173.png)
![Dock](https://user-images.githubusercontent.com/45864985/127258256-45678ffc-057f-40cc-891a-aaf437b1948f.png)
